packages/core/src/Stores/traders-hub-store.js

```diff
@@ -413,6 +418,18 @@ export default class TradersHubStore extends BaseStore {
             account => account.platform === CFD_PLATFORMS.DXTRADE
         );
     }
+    getAvailableCTraderAccounts() {
+        if (this.is_eu_user && !this.is_demo_low_risk) {
+            this.available_ctrader_accounts = this.available_cfd_accounts.filter(
+                account => account.platform === CFD_PLATFORMS.CTRADER
+            );
+            return;
+        }
+        this.available_ctrader_accounts = this.available_cfd_accounts.filter(
+            account => account.platform === CFD_PLATFORMS.CTRADER
+        );
+    }
+
```

Comment:
@hirad-deriv a little bit confused regarding `getAvailableCTraderAccounts` function. seems like in any case you do the same thing. 🤔

Reply:
```suggestion
    getAvailableCTraderAccounts() {
        this.available_ctrader_accounts = this.available_cfd_accounts.filter(
            account => account.platform === CFD_PLATFORMS.CTRADER
        );
    }

```

Reply:
Hi @george-usynin-deriv, We will have a refinement for this.